support-frontend/app/views/testUsers.scala.html

```diff
@@ -10,7 +10,7 @@
         <title>Test Users</title>
     </head>
     <body>
-        <h1>New Test User key: @key</h1>
+        <h1>New Test GetUser key: @key</h1>
```

Comment:
intellij isn't so brilliant at renaming variables in JS code... 🙈

Reply:
No it's very poor